app/assets/stylesheets/nav.scss

```diff
@@ -0,0 +1,23 @@
+header {
+  background: silver;
+  padding: 10px;
+  overflow: hidden;
+  ul {
+    margin: 0;
+    padding: 0;
+    list-style-type: none;
+  }
+  a {
+    text-decoration: none;
+    font-weight: bold;
+    color: white;
+  }
+}
+
+.left-nav {
+  float: left;
+}
+
+.right-nav {
+  float: right;
+}
```

Comment:
not actual styles, I just wanted to put something here to make sure scss compiling was working.